### ISSUE TITLE
Invalid storage of attributes

### ISSUE THREAD

**Issue by kdzwinel** [jigoshop/jigoshop#108]
After fixing #104 I started digging into the way that variables are stored. Original programmer of configurable products used `'tax_' . sanitize_title($attribute_name)` instead of attribute slug or id. That may cause data corruption (as ID created that way is not guaranteed to be unique) but also makes it impossible to retrieve attribute value names (ie. 'black & white' for attribute 'color'). Function `get_term_by` needs taxonomy id not some custom string (`'tax_' . sanitize_title($attribute_name)`) to retrieve term details. That is why slugs of options are displayed everywhere instead of their names. 

tl;dr all `'tax_' . sanitize_title($attribute_name)` must go before 1.0
